### PR TITLE
lilo: init at 24.2

### PR DESCRIPTION
**Pull request by KAction**
###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [ ] NixOS
   - [ ] macOS
   - [x] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [x] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [21.11 Release Notes (or backporting 21.05 Relase notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2111-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/tools/misc/lilo/default.nix [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+stdenv.mkDerivation rec {
+  pname = "lilo";
+  version = "24.2";
+  src = fetchurl {
+    url = "https://www.joonet.de/lilo/ftp/sources/${pname}-${version}.tar.gz";
+    hash = "sha256-4VjxneRWDJNevgUHwht5v/F2GLkjDYB2/oxf/5/b1bE=";
+  };
+  nativeBuildInputs = [ dev86 sharutils ];
+  DESTDIR = builtins.placeholder "out";
```
Comment (IvarWithoutBones) [inline pkgs/tools/misc/lilo/default.nix:11]: This is already in scope, I think.
```suggestion
  DESTDIR = placeholder "out";
```

**Review by IvarWithoutBones — CHANGES_REQUESTED** [review]
Just 1 minor nit, otherwise LGTM :+1: 

Would also be nice if you could run `nixpkgs-fmt` on this, to make formatting more consistent with the rest of the codebase :)

**Comment by KAction** [comment]
```
>>= nixpkgs-fmt  pkgs/tools/misc/lilo/default.nix
0 / 1 have been reformatted
```

I think `nixpkgs-fmt` is already happy.

File: pkgs/tools/misc/lilo/default.nix (hunk 1 above)
Comment (KAction) [reply pkgs/tools/misc/lilo/default.nix]: Applied.

**Pull request merged**